app/assets/stylesheets/sass/views/attendances/_index.css.scss

```diff
@@ -126,4 +133,43 @@
     width: 30%;
   }
 }
- 
\ No newline at end of file
+
+.table-item {
+  width: 40rem;
+  
+  tbody {
+    height: 60vh;
+    overflow: auto;
+    width: 100%;
+
+    tr:nth-child(1) {
+      width: 200px;
+    }
+  }
+}
+
+.table-wrapper {
+  position:relative;
+}
+.table-scroll {
+  max-height:60rem;
+  overflow:auto;
+}
+.table-wrapper table {
+  width:100%;
+}
+
```

Comment:
left an empty line between classes. also a space after the `colon` 

```
position: relative;
```
not 
```
position:relative;
```
check the whole file

Reply:
Change made, file was fixed